src/components/AccountInline.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <span class="account-inline">
+    <AeIdenticon :address="address" />
+    <template v-if="name">
+      {{ name }}
+    </template>
+    <AeAddress
+      v-else
+      :address="address"
+      length="short"
+    />
+  </span>
+</template>
+
+<script>
+import { AeIdenticon } from '@aeternity/aepp-components-3';
+import AeAddress from './AeAddress.vue';
+
+export default {
+  components: { AeIdenticon, AeAddress },
+  props: {
+    address: { type: String, required: true },
+  },
+  computed: {
+    name() {
+      return (
+        [...this.$store.getters.identities, ...this.$store.state.addressBook]
+          .find(({ address }) => address === this.address) || { name: '' }
+      ).name;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.account-inline .ae-identicon {
+  vertical-align: middle;
+  height: 1.055em;
```

Comment:
The idea is to make an identicon size relative to font size.

|                 | Font size | Icon height | Designs |
|----------|----------|------------ | --------|
| Mobile    | 23px        | 24px            | [link](https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5c586d1e9022ce4290ddf35c) |
| Desktop  | 30px       | 32px            | [link](https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5c5c437dedd60b051b93df4c) |

(24/23 + 32/30) / 2 = 1,055072463768116 ~ 1,055